pkg/volume/local/local.go

```diff
@@ -204,8 +210,18 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		glog.Errorf("cannot validate mount point: %s %v", dir, err)
 		return err
 	}
-	if !notMnt {
```

Comment:
I think we still need the notMnt check.  It's checking on `dir`, which is the pod's path, not the global path.  It's to make sure we don't mount the volume twice to the same pod if Mount gets called again.